src/pages/index.tsx

```diff
@@ -1,27 +1,60 @@
+import axios, { AxiosError } from "axios";
 import CtaSection from "../components/cta";
 import FooterSection from "../components/footer";
 import Header from "../components/header";
 import MissionSection from "../components/mission";
-import ModProfiles from "../components/mod-profiles";
-import { Container, Divider } from "@chakra-ui/react";
+import Stats from "../components/stats/index";
+import Team from "../components/team";
+import { StatsData } from "../utils/interfaces";
 
-function IndexPage() {
+
+interface Props {
+  statsData: StatsData
+}
+
+function IndexPage(props: Props) {
+  const { statsData } = props;
   return (
-    <Container
-      maxWidth={{ base: "full", md: "container.lg" }}
-      marginTop={{ base: 12, md: 28 }}
-      marginBottom={{ base: 8, md: 12 }}
-    >
+    <main>
       <Header />
-      <Divider marginY={20} />
       <MissionSection />
-      <Divider marginY={20} />
-      <ModProfiles />
-      <Divider marginY={20} />
+      <Stats data={statsData} />
+      <Team />
       <CtaSection />
       <FooterSection />
-    </Container>
+    </main>
   );
 }
 
+export async function getServerSideProps(context: any) {
```

Comment:
Also I think rather than use some hacks & "uneasy code" (because `getServerSideProps` will unnecessarily increase the TTL of this `index.tsx` file), we can use [SWR](https://swr.vercel.app). That way not only can we share the latest information about the community but keep load times of the page snappy as well.

So each `page` component in a Next.js project that uses [`getServerSideProps` renders the page on request](https://nextjs.org/docs/basic-features/data-fetching/get-server-side-props#when-should-i-use-getserversideprops) & not statically! I say its better to serve a single static page (i.e a HTML page) & let React do the heavy-lifting of client-side data fetching using `useSWR`.

This way our single page landing page will have a lower TTL (which directly equates to better SEO) & better cache control of the data on client-side rather than on server-side.